test/routes/index.spec.unit.ts

```diff
@@ -0,0 +1,24 @@
+import app from "../../src/app";
+import * as request from "supertest";
+
+describe("Basic URL Tests", () => {
+
+  it("should find start page", async () => {
+    const response = await request(app)
+      .get("/promise-to-file");
+    expect(response.status).toEqual(200);
+  });
+
+  it("should find company number page", async () => {
+    const response = await request(app)
+      .get("/promise-to-file/company-number")
+      .set("Referer", "/")
```

Comment:
Referer not used at this point. Might not be needed if we can find a way of capturing the page being requested via middleware for the back button.

Reply:
Will remove 'Refererer' as not yet used or needed by this skeleton app.